### DISCUSSION TITLE
Email templates: Dependent fields

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#14347] [Feature improvement]
What means "Dependent fields"? It contains a lot of confusing variables. What is responsible to show fields vars here? Can it be controlled somehow? Also "Fields from dependent modules" those variables doesn't returns any results.  
Maybe I missed something? Any guide is available? 

Thank you

**Comment by vovpff** [comment]
Something wrong with your system. There are should be pickists of fields from current an related modules
